packages/lib/src/vis/shared/hooks.ts

```diff
@@ -163,3 +163,16 @@ export function useCanvasEvents(callbacks: CanvasEventCallbacks) {
     };
   });
 }
+
+export function useWorldToHtml(): (point: Vector3) => Vector3 {
```

Comment:
Two suggestions:

- Provide this through context, like `worldToData` and `dataToWorld`.
- Inline `projctCameraToHtml` in the `useCallback` below since it should now only be used there.